### ISSUE TITLE
Enable staking and set initial stake amount for testnet

### ISSUE THREAD

**Issue by JulissaDantes** [recallnet/ipc#141]
The selected amount must satisfy the following criteria:

`Ms = Ca(Tv * 2/3)`

where `Tv * 2/3` indicates the number of validators required to execute an attack. Typically, this number is chosen as a rule of thumb.

Note: By default, staking will only work with the native token of the parent settlement chain (e.g., ETH or FIL).  There is a separate ticket to enable staking in our own custom subnet token https://github.com/amazingdatamachine/ipc/issues/150 

May be blocked by: https://github.com/amazingdatamachine/ipc/issues/124
